src/Veldrid/D3D11/D3D11TextureView.cs

```diff
@@ -71,8 +71,10 @@ public D3D11TextureView(D3D11GraphicsDevice gd, ref TextureViewDescription descr
                 {
                     uavDesc.ViewDimension = UnorderedAccessViewDimension.Texture3D;
                     uavDesc.Texture3D.MipSlice = (int)description.BaseMipLevel;
-                    uavDesc.Texture3D.FirstWSlice = (int)description.BaseArrayLayer;
-                    uavDesc.Texture3D.WSize = (int)description.ArrayLayers;
+
+                    // Map the entire range of the 3D texture.
+                    uavDesc.Texture3D.FirstWSlice = 0;
+                    uavDesc.Texture3D.WSize = (int)d3dTex.Depth;
```

Comment:
The actual fix.  ArrayLayers is always going to be 1 for 3D textures, right?  I also think we want to map the whole 3D texture instead of starting the mapping at BaseArrayLayers.

Reply:
The interface in Veldrid isn't super clear about how this is going to behave, and it's enough of an edge case that I haven't thought about it much. I think it's reasonable to treat the depth of a 3D texture as separate from the idea of an "array layer" although they are somewhat synonymous in other contexts. I think it's more natural to pass (0, 1) as (base array layer, layer count) than it would be to pass (0, depth). So this fix looks reasonable to me.